.eslintrc.js

```diff
@@ -15,5 +15,6 @@ module.exports = {
   rules: {
     "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+    "@typescript-eslint/ban-types": "off",
```

Comment:
To avoid this:
![image](https://user-images.githubusercontent.com/33035102/190918484-1e969b08-1d52-4915-866d-b632a11fda21.png)